### ISSUE TITLE
Screen reader does not announce descriptive information for graph slider.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#24368]
#### Expected behaviour
The screen reader should provide descriptive and contextual information for the graph slider. For example:   “Performance graph range selected from June 1st to June 15th”.
>This should include associated dates, selection summaries, and any other relevant context to help users understand what the slider controls.

#### Actual behaviour
Screen reader is not announcing descriptive information of slider controls. It is announcing it as 'Start percentage 90/End percentage 96'. 

https://github.com/user-attachments/assets/5c306e79-8eb4-4c0c-be41-da04d0aecdb5

#### Live demo with steps to reproduce

1. Open URL: [Flags marking events Demo | Highcharts](https://www.highcharts.com/demo/stock/flags-general) in Edge Browser.
2. Press tab key to reach 'Graph slider' present below the graph.
3. Verify using tab key whether screen reader announces descriptive information of slider control or not. 

#### Product version
highcharts": "11.4.8",
highcharts-react-official": "3.2.3

#### Affected browser(s)
OS: Windows 11 Enterprise Insider Preview 25H2 26300.7877
Browser: Microsoft Edge Version 146.0.3856.62
Screen Reader: Narrator
Same issue in Chrome and other Screen readers like NVDA and JAWS as well.

**Comment by msftedad** [comment]
Hi Team, Any update here?
